### ISSUE TITLE
layers.Masking causes model.fit much slower than not using this layer

### ISSUE THREAD

**Issue by tinyxuyan** [keras-team/keras#17076]
When I use layers.Masking, the model is trained much slower than not using case. The example code is given as below:

    RxInputSignal = keras.Input(shape=[72, 2, 256])
    xx = layers.Permute((1, 3, 2))(RxInputSignal)
    xx = layers.Reshape((72, 512))(xx)
    xx = layers.Masking(mask_value=0.)(xx)
    
    xS = layers.LayerNormalization()(xx)
    xS = layers.Bidirectional(layers.LSTM(768, return_sequences=True,
                                          dropout=0.0))(xS)

version: tensorflow 2.4.0 
python 3.6.9

**Comment by NicoRixe** [comment]
Hey,
I have the same problem. Found a solution for that? Thanks!
